### DISCUSSION TITLE
Cannot log-in with the PRIDE Submission tool

### DISCUSSION THREAD

**Discussion by ypriverol** [PRIDE-Archive/pride-resources#7] [Q&A]
Please confirm that you can log into the main PRIDE website with your account: [https://www.ebi.ac.uk/pride/login](https://www.ebi.ac.uk/pride/login) Perhaps you can try changing your password, and then trying again.

**Comment by enryH** [comment]
Is the url still up to date? I get an 404 Error...

**Reply by chakrabandla** [reply]
Sorry updated the URL. Here is the correct one: https://www.ebi.ac.uk/pride/login
